packages/manager/apps/web/client/app/hosting/database/DATABASE_LIST.html

```diff
@@ -61,8 +61,7 @@
                             <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
                         </button>
                         <ul class="dropdown-menu m-0 p-0">
-                            <li data-uib-tooltip="{{:: 'hosting_tab_DATABASES_configuration_create_step1_no_db_left' | translate }}"
-                                data-tooltip-enable="!ctrlTabDatabases.canCreateDatabase">
+                            <li>
```

Comment:
Why remove it ? Was it not making sense anymore ?

Reply:
It says "You don't have any databases available" on a disabled button to` create database`. So removed it.

Reply:
I think we should avoid removing that warning. It help our customers to understand why they can’t create database :/

If it’s here, there is a reason :/ -> We should avoid updating this behaviour in this PR, it’s not a goal of your PR :)

Reply:
Have added this back. May be we can remove the tooltips in another PR :)